mobify/bellows
# Release v6.0.0

## Changes
- NPM release
- Remove Zepto support in favor of jQuery
- Migrate all dependencies to NPM, remove Bower
- Migrate JS linting to eslint
- Migrate SCSS compilation to libsass
- Lint SCSS with sass-lint
- Update circleCI config with modern testing steps
- Add some GitHub templates for contributing, issues, and PRs

## How to Test
- Checkout code
- Review package.json
- Review Changelog
- `npm install`
- `grunt test` or `grunt test:browser`
- Check examples with `grunt examples`

## Todos:
- [x] Add an `.npmignore`
- [ ] Post merge: Publish to NPM
